corehq/apps/app_manager/templates/app_manager/apps_base.html

```diff
@@ -33,15 +33,33 @@
       actually used in app manager (no tooltip).
   -->
   {% compress js %}
-    <script src="{% static 'jquery-ui/ui/minified/core.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/widget.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/mouse.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/position.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/autocomplete.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/draggable.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/droppable.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/menu.min.js' %}"></script>
-    <script src="{% static 'jquery-ui/ui/minified/sortable.min.js' %}"></script>
+    <!-- this defines jquery ui ($.ui) so this MUST come first -->
```

Comment:
Can you make sure QA knows to test this doesn't re-introduce the tooltip issue described in the comment above? That comment was added in https://github.com/dimagi/commcare-hq/pull/9973/commits/1bc4f4dc55660d220f3e2c0c4783251537f84ceb during the Bootstrap 3 migration. The PR description notes that it fixes "Tooltips don't work (e.g., hovering over the pencil icons in the menu)" so I think that's what should be tested.

...I forgot how much work the B3 migration was. 😅

Reply:
what's the tooltip issue? I think i'm not seeing that comment? 👀

Reply:
nm I think you meant this

>jQuery UI needs to be included before Bootstrap's JavaScript, otherwise the two tooltip widgets conflict. The B3 base template takes care of that when you use the @use_jquery_ui decorator, but app manager doesn't, so instead include only the pieces actually used in app manager (no tooltip).

The order didn't change here between bootstrap and Jquery UI. I included only what was originally in `core.js` here...nothing extra. `core.js` is no longer usable in this case as it requires something like `requirejs` in order to load.

Reply:
okay, cool